mavros_msgs/PositionTarget goal_pos
---
float64 dist